computeengine/mailjet.js

```diff
@@ -18,7 +18,8 @@ var mailer = require('nodemailer');
 var smtp = require('nodemailer-smtp-transport');
 
 var transport = mailer.createTransport(smtp({
-  service: 'Mailjet',
+  host: "in.mailjet.com",
```

Comment:
Can you please change this from double quotes to single quotes? The linting will not pass unless this uses single quotes.